msfits/MSFits/MSFitsOutput.cc

```diff
@@ -1756,42 +1755,10 @@ Bool MSFitsOutput::writeAN(FitsOutput *output, const MeasurementSet &ms,
         // Set the ones we're not going to change once
         *orbparm = 0.0;
         *poltya = " ";
-        *polaa = 0.0;
```

Comment:
what will it now write when it can't find a polarization type? I assume this should not happen, but as the function just warns and continues would that result in uninitialized data?